app/views/ems_container/_show_dashboard.html

```diff
@@ -1,38 +1,83 @@
-<div ng-app="containerDashboard" ng-controller="singleProviderDashboardController as dashboard" class="container-fluid container-tiles-pf containers-dashboard miq-dashboard-view single-provider-containers-dashboard">
+<div ng-app="containerDashboard" ng-controller="containerDashboardController as dashboard" class="container-fluid container-tiles-pf single-provider-containers-dashboard miq-dashboard-view">
     <div class="row row-tile-pf" >
         <div class="col-xs-6 col-sm-4 col-md-2">
             <div pf-card class="col-xs-12 provider-card" show-top-border="true">
-                <span class="provider-icon-large {{providerTypeIconClass}}"></span>
+                <span ng-if='!isAtomic' class="provider-icon-large {{providerTypeIconClass}}"></span>
+                <span ng-if='isAtomic' class="atomic-large"><img src='/images/icons/new/vendor-atomic_enterprise.png' ></span>
             </div>
         </div>
         <div class="col-xs-12 col-sm-12 col-md-10">
             <div class="row">
                 <div class="col-xs-6 col-sm-6 col-md-3">
-                    <div  pf-aggregate-status-card status="objectStatus.nodes" url="navigation" show-top-border="true" layout="mini" class="card-pf-title-container-dash"></div>
+                    <div  pf-aggregate-status-card status="objectStatus.nodes" url="navigation" show-top-border="true" layout="mini"></div>
                 </div>
                 <div class="col-xs-6 col-sm-6 col-md-3">
-                    <div  pf-aggregate-status-card status="objectStatus.containers" url="navigation" show-top-border="true" layout="mini" class="card-pf-title-container-dash"></div>
+                    <div  pf-aggregate-status-card status="objectStatus.containers" url="navigation" show-top-border="true" layout="mini"></div>
                 </div>
                 <div class="col-xs-6 col-sm-6 col-md-3">
-                    <div  pf-aggregate-status-card status="objectStatus.registries" url="navigation" show-top-border="true" layout="mini" class="card-pf-title-container-dash"></div>
+                    <div  pf-aggregate-status-card status="objectStatus.registries" url="navigation" show-top-border="true" layout="mini"></div>
                 </div>
                 <div class="col-xs-6 col-sm-6 col-md-3">
-                    <div  pf-aggregate-status-card status="objectStatus.projects" url="navigation" show-top-border="true" layout="mini" class="card-pf-title-container-dash" ></div>
```

Comment:
Html copied over from overview page to be uniform.
The single provider page wasn't updated for a while now.